LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2020 Flash0ver
```

Comment:
Is this still correct as you committed this change on the last day of last year, or do we need to update this now for 2021? 😛

Reply:
I've asked myself that same question and came across this post, advising to use the original year when the copyright was applied: https://opensource.stackexchange.com/a/1523